app/src/main/java/com/iic/shopingo/ui/trip_flow/activities/ManageTripActivity.java

```diff
@@ -92,11 +98,11 @@ public int getCount() {
   }
 
   @Override
-  public void onRequestAccepted(Request request) {
+  public void onRequestAccepted(IncomingRequest request) {
     ShoppingList sl = new ShoppingList();
-    sl.requesterName = request.name;
-    sl.phoneNumber = request.location.phone;
-    for (String itemTitle : request.items) {
+    sl.requesterName = request.getRequester().getFirstName();
+    sl.phoneNumber = request.getRequester().getPhoneNumber();
```

Comment:
I imagine it heavily depends on Dalvik/ART.
Technically, it just needs to load a pointer, which should already be in cache for the 2nd line, and the runtime should JIT the whole thing to do just that.
What it actually does, I have no idea.
I imagine you'd want a local variable for readability if you have to do it more than a few times.